app/services/login/user_verifier.rb

```diff
@@ -137,6 +138,17 @@ def attempt_secondary_idme_identifier
       Rails.logger.info("[Login::UserVerifier] Attempting alternate type=#{type}  identifier=#{identifier}")
     end
 
+    def validate_linked_user_verifications
+      linked_user_verifications = UserVerification.where(user_account_id: existing_user_account.id)
```

Comment:
Easy way to do this is:
`UserVerification.where(user_account: existing_user_account).any?(&:locked)`

Reply:
I updated this with `any` but used a modified evaluation block because we only want to check for locked UserVerifications of the same CSP type.

`UserVerification.where(user_account: existing_user_account).any? { |v| v.send(type).present? && v.locked }`